plugin_interpreter/src/rethink_interface.py

```diff
@@ -26,28 +25,40 @@ class RethinkInterface:
     It runs as a process and is instantiated by and controlled by the
     SupervisorController class.
     """
+    VALID_STATES = ["Ready", "Pending", "Done", "Error", "Stopped", "Waiting"]
```

Comment:
docs include state "Active" and "Preparing"

probably don't need to include Preparing here